core/reporter-api/src/main/java/org/arquillian/reporter/api/model/entry/KeyValueEntry.java

```diff
@@ -57,6 +57,7 @@ public KeyValueEntry(String key, String value) {
         this.value = new StringEntry(value);
     }
 
+
```

Comment:
Please, clean the PR so it doesn't contain any formatting changes in files that haven't been changed